docker/luna_tutorial/dockerfile_girder

```diff
@@ -42,4 +43,4 @@ ENTRYPOINT ["/usr/bin/tini", "--"]
 
 RUN mkdir /logs && chmod -R 777 /logs
 USER $USER_NAME
-CMD python vmount/provision.py --sample-data && girder serve --database mongodb://mongodb:27017/girder
```

Comment:
A comment on provision.py could be useful here:
- Creates an admin user (admin/password)
- Downloads data from https://data.kitware.com/#user/61b9f3dc4acac99f42ca7678/folder/61b9f4564acac99f42ca7692
and saves the slides at ~/vmount/PRO_12-123/data/toy_data_set
- Creates an assetstore at ~/vmount/assetstore
- Organizes the sample images in TCGA collection
- Uploads annotations to the slides.